beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -374,6 +374,14 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         if slot == head_state.slot {
             Ok(head_state)
         } else if slot > head_state.slot {
+            if slot > head_state.slot + 3 {
```

Comment:
I have a preference for `3` to be defined as a constant at the top of the file, especially because it appears again in the warn log.

Reply:
Of course! You're entirely right, thank you for pointing this out!